package.json

```diff
@@ -652,57 +652,57 @@
     },
     "configuration": {
       "type": "object",
-      "title": "Azure IoT Hub Toolkit configuration",
+      "title": "%azure-iot-toolkit.configuration.title%",
       "properties": {
         "azure-iot-toolkit.deviceConnectionString": {
           "type": "string",
           "default": "<<insert your Device Connection String>>",
-          "description": "Device Connection String"
+          "description": "%azure-iot-toolkit.deviceConnectionString.description%"
```

Comment:
For configuration, I'd prefer to be `azure-iot-toolkit.configuration.deviceConnectionString.description`

Reply:
Okay. I will change it.